### PR TITLE
[tutorials] Move pyroot tutorials

### PR DESCRIPTION
**Pull request by mdessole**
# This Pull request:

is a continuation of https://github.com/root-project/root/pull/17102

`PyRoot` tutorial category is being deprecated: its content is being distributed across other categories, or integrated in the documentation when no candidate category was identified.

### CODE REVIEW

File: bindings/pyroot/pythonizations/python/ROOT/_pythonization/__init__.py [hunk 1]
```diff
@@ -55,6 +55,258 @@ def pythonization(class_name, ns='::', is_prefix=False):
     Returns:
         function: function that receives the user-defined function and
             decorates it.
+
+    ### Pythonization example
```
Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/_pythonization/__init__.py:59]: I think these two tutorials make the docstring of this decorator far too long. What I would propose is to add the text to the newly created Python Interface documentation page instead, i.e. https://root.cern/doc/master/group__Pythonizations.html , found at `bindings/pyroot/pythonizations/doc`.

File: tutorials/pyroot/ratioplot.py
Comment (vepadulano) [inline tutorials/pyroot/ratioplot.py:1]: I can't find this tutorial anywhere else, where was it moved?

**Review by vepadulano — CHANGES_REQUESTED** [review]
Thank you! See a couple of comments

File: bindings/pyroot/pythonizations/python/ROOT/_pythonization/__init__.py (hunk 1 above)
Comment (mdessole) [reply bindings/pyroot/pythonizations/python/ROOT/_pythonization/__init__.py]: I agree, I moved the text where you suggested. Thanks!

File: tutorials/pyroot/ratioplot.py
Comment (mdessole) [reply tutorials/pyroot/ratioplot.py:1]: It has been deleted, as it does not use `TRatioPlot`. The corresponding C macro is named `ratioplotOld.C` and I quote it "For ROOT version >= 6.08 TRatioPlot should be used.". Should we move to legacy (together with `ratioplotOld.C`) instead?

Comment (mdessole) [reply tutorials/pyroot/ratioplot.py:1]: @vepadulano ping

**Review by vepadulano — APPROVED** [review]
Thank you so much! This is a great improvement in the organization of our tutorials 🚀

**Review by martamaja10 — APPROVED** [review]
Thank you Monica and Declan for all the work :)

**Pull request merged**